src/atoms/SvgIcon/CrossedVideoCamera.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const CrossedVideoCamera = props =>
+	(
+		<svg xmlns="http://www.w3.org/2000/svg" width="52" height="45" viewBox="0 0 52 45">
```

Comment:
I think we should use 100% H and W, so we can take effect on this Icon only by the parent container.